engine/templates/layout.go.html

```diff
@@ -30,6 +30,23 @@
                     <img src="img/logo-pdok.svg" alt="Logo PDOK: Ga naar de landing page">
                 </a>
             </div>
+            <div class="container">
+                <ol class="breadcrumb">
+                    <li class="breadcrumb-item">Datasets</li>
+                    <li class="breadcrumb-item"><a href="{{ .Config.BaseURL }}">{{ .Config.ShortTitle }}</a></li>
+                    {{ range $breadcrumb := .Breadcrumbs }}
+                    <li class="breadcrumb-item"><a href="{{ $breadcrumb.Path }}">{{ $breadcrumb.Name }}</a></li>
+                    {{ end }}
+                </ol>
+                <ol class="breadcrumb" style="float: right;">
```

Comment:
Even kijken of we dit anders kunnen stylen, float is niet echt heel nice. Maar kan wat mij betreft later

Reply:
Ja, eens, dit is nogal een hack om het voor elkaar te krijgen, zeker op de lijst voor latere verbetering